docs/superpowers/specs/assets/2026-06-18-mobile-ux/mockup.html

```
@@ -1,309 +1,895 @@
+          <span class="lab">Read-only</span>
+          <span class="cap" style="flex: 1">Lock the layout for viewing</span>
+          <div class="lockswitch"><span class="knob"></span></div>
+        </div>
+        <div class="vrow tall">
+          <span class="lab">Zoom</span>
+          <div class="stepper">
+            <div class="pm">&#8722;</div>
+            <div class="val">100%</div>
+            <div class="pm">&#43;</div>
+          </div>
+          <button class="fitinline">&#10530; Fit to screen</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)